src/_shared/scss/_adverts-brand.scss

```diff
@@ -1,12 +1,17 @@
+.adverts--tone-brands .button--large {
```

Comment:
So you already have it here, you shouldn't need to add it in all the other styles...

Reply:
As above.